src/shared/pal/src/thread/process.cpp

```diff
@@ -2431,7 +2431,7 @@ CreateProcessModules(
         int devHi, devLo, inode;
         char moduleName[PATH_MAX];
 
-        if (sscanf_s(line, "%p-%p %*[-rwxsp] %p %x:%x %d %s\n", &startAddress, &endAddress, &offset, &devHi, &devLo, &inode, moduleName, ARRAY_SIZE(moduleName)) == 7)
+        if (sscanf_s(line, "%p-%p %*[-rwxsp] %p %x:%x %d %[^\n]", &startAddress, &endAddress, &offset, &devHi, &devLo, &inode, moduleName, ARRAY_SIZE(moduleName)) == 7)
```

Comment:
I'm not sure if `%[^\n]\n` is any different than what I ended up with.  Yes, it is to include file/directory names with blanks.

The change will be in an upcoming runtime PR to the createdump code. I'm not going to make it to the runtime PAL code because it will go away when dbgshim and the support code, etc. goes away.

Reply:
I made the change to %[^\n]\n